### PR TITLE
Simple Weather App by Amanda :) 

### PR DESCRIPTION
**Pull request by amandaflod**
Link to my deployed project:

https://elastic-blackwell-e78f71.netlify.app/

My reflections on how this week's project turned out:

I think it turned out quite well in the end, but it was a long way to get there and a lot of getting stuck on the way. I was too ashamed of my code to commit any changes earlier than today, as even I could not make sense of what I had written. But I think that I have learned a lot this week, especially when it comes to cutting down on the code and trying to make it as readable and clean as possible!

Things I'd like to have clarified or explained in more detail:
Nothing I can think of right now :)

### CODE REVIEW

**Review by lindahz — APPROVED** [review]
Amanda, I am so impressed by your code! 😍  It was really hard to give you any constructive feedback, especially since you managed to keep your code SUPER clean and structured. I love the clean design, colors and the nice gradient background you picked.

Your javascript was also a bit different from mine, and I am happy I got the chance to learn some other ways of writing the code. I can see that you spent time to really understand the code, because it looks well thought out.

 I only have a few comments as you can see.

Great job!!! ❤️⭐️

File: script.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+fetch(apiUrl)
+  .then(response => response.json())
+  .then(data => {
+    renderWeather(data)
+    renderForecast(data)
+  })
+  .catch(error => {
+    console.error(error)
```
Comment (EmOlsz) [inline script.js:19]: Great job with `console.error()`!

File: script.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  const sunriseTimeString = sunriseDate.toLocaleTimeString([], options)
+  const sunsetTimeString = sunsetDate.toLocaleTimeString([], options)
+  sunriseElement.innerHTML = `Sunrise: ${sunriseTimeString}`
+  sunsetElement.innerHTML = `Sunset: ${sunsetTimeString}`
+}
+
+const formatDay = day => {
+  const forecastDate = new Date(day * 1000)
+  const forecastDay = forecastDate.toString()
+  shortForecastDay = forecastDay.substring(0, 3)
```
Comment (EmOlsz) [inline script.js:63]: You didn't use `const`, `let` or `var` keyword and by mistake you created global variable - this is huge antipattern, please fix that

File: style.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    padding: 100px 360px 60px 360px;
+  }
+  .weather-container ul {
+    justify-content: space-evenly;
+  }
+  .forecast-container {
+    display: flex; 
+    justify-content: space-evenly; 
+  }
+  #forecastList {
```
[the remark this answers is not in the document]
Comment (EmOlsz) [reply style.css:87]: Even more - we should use only classes, no id's

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Good job Amanda! Your project is very clean, you also use a lot of good practices such as lack of parentheses around one argument in function declaration, `catch()` method in fetch and shorthand for arrow functions. Congratulations :)
I've added few comments, please provide changes according to them

**Pull request closed without merging**